pod/perlfunc.pod

```diff
@@ -4195,7 +4195,7 @@ follows:
     my ($sec,$min,$hour,$mday,$mon,$year,$wday,$yday,$isdst) =
                                                 localtime(time);
 
-All list elements are numeric and come straight out of the C `struct
+All list elements are numeric and come straight out of the C 'struct
```

Comment:
C<struct tm> sounds better IMHO

Reply:
This is my gentle intro to contributing :-) I'm planning to review more docs, and will change this on my next pull request.